images/macos/software-report/SoftwareReport.Toolcache.psm1

```diff
@@ -44,6 +44,14 @@ function Get-ToolcacheGoTable {
     return $Content
 }
 
+function Get-CodeQLBundleVersion {
+    $CodeQLVersionsWildcard = Join-Path $Env:AGENT_TOOLSDIRECTORY -ChildPath "CodeQL" | Join-Path -ChildPath "*"
+    $CodeQLVersionPath = Get-ChildItem $CodeQLVersionsWildcard | Select-Object -First 1 -Expand FullName
+    $CodeQLPath = Join-Path $CodeQLVersionPath -ChildPath "x64" | Join-Path -ChildPath "codeql" | Join-Path -ChildPath "codeql"
+    $CodeQLVersion = & $CodeQLPath version --quiet
+    return "CodeQL Action Bundle $CodeQLVersion"
```

Comment:
If we are planning to use multiple version it should be like:

CodeQL Action Bundle:
- 0.0.0.1
- 0.0.0.2
- etc